### PR TITLE
Described deployment of first application

### PR DESCRIPTION
**Pull request by dbecvarik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +57,112 @@ Creating machine...
...
+$  kubectl proxy
+```
+
+Than you can access your application via following command:
+```
+curl http://localhost:8001/api/v1/namespaces/default/services/hello/proxy/
+```
+
+### Service
+Did you noted a world `service` inside the url? Yes, we are using kubernetes service to access our application. You can see how the service object looks by running:
```
Comment (jwerak) [inline README.md]: Did you noted -> Did you note

File: README.md [hunk 2]
```diff
@@ -58,6 +57,112 @@ Creating machine...
...
+```
+$  kubectl describe service hello
+```
+
+Or you can list all other service by:
```
Comment (jwerak) [inline README.md]: service -> services

File: README.md [hunk 3]
```diff
@@ -58,6 +57,112 @@ Creating machine...
...
+```
+$  kubectl get services
+```
+
+Why service is there? The service is very important object in Kubernetes. Its an abstraction for your deployed applications. It helps you to discover your services, to load balance them and
```
Comment (jwerak) [inline README.md]: It helps you to discover your services 
Service helps discover services, it makes sense, but can be confusing, I would use instead
It helps discover your Pods ||  It helps discover your application instances (pods).

**Comment by dbecvarik** [comment]
added persistent volumes. This should be ok to review now

File: README.md [hunk 4]
```diff
@@ -76,25 +116,40 @@ To access your app we will hook inside Kubernetes network via `kubectl proxy` co
...
+! IMPORTANT - Do not stop proxy as following commands will not work.
 
 Than you can access your application via following command:
 ```
 curl http://localhost:8001/api/v1/namespaces/default/services/hello/proxy/
 ```
 
 ### Service
-Did you noted a world `service` inside the url? Yes, we are using kubernetes service to access our application. You can see how the service object looks by running:
+Did you note a world `service` inside the url? Yes, we are using kubernetes service to access our application. You can see how the service object looks by running:
```
Comment (jwerak) [inline README.md]: world -> word

File: README.md [hunk 5]
```diff
@@ -56,7 +54,266 @@ Starting local Kubernetes cluster...
...
+        terminationMessagePath: /dev/termination-log
+        terminationMessagePolicy: File
+        volumeMounts:
+        - mountPath: /srv/
+          name: test
+      volumes:
+      - name: test
+        persistentVolumeClaim:
+          claimName: testpvc
+	  dnsPolicy: ClusterFirst
```
Comment (jwerak) [inline README.md]: bad indent?

**Comment by dbecvarik** [comment]
Thanks for review.

**Pull request merged**